inference-engine/thirdparty/clDNN/api/intel_gpu/runtime/debug_configuration.hpp

```diff
@@ -34,8 +35,12 @@ class debug_configuration {
     std::string dump_layers;        // Dump intermediate buffers of specified layers only, separated by space
     std::string dry_run_path;       // Dry run and serialize execution graph into the specified path
     int dump_layers_dst_only;       // Dump only output of layers
+    int dump_layers_limit_batch;    // Limit the size of batch to dump
     int base_batch_for_memory_estimation; // Base batch size to be used in memory estimation
     static const debug_configuration *get_instance();
+
+private:
+    void print_help_messages();
```

Comment:
What about making it as static function to reduce the amount of rebuild?

Reply:
I changed this function to static function.